src/crimson/os/seastore/cache.h

```diff
@@ -978,7 +978,8 @@ class Cache {
     auto result = epm.alloc_new_non_data_extent(t, T::TYPE, length, hint, gen);
 #endif
     if (!result) {
-      return nullptr;
+      SUBERRORT(seastore_cache, "insufficient space", t);
+      std::rethrow_exception(crimson::ct_error::enospc::exception_ptr());
```

Comment:
Will the exception be converted into errors in the errorator?

Reply:
yes